src/raster/r.niche.similarity/r.niche.similarity.py

```diff
@@ -219,7 +218,7 @@ def main(options, flags):
 
     # Write results to csv file
     if OPF:
-        IND = [["Statistic", "Layer 1", "Layer 2", "value"]] + Dind + Iind + Cind
+        IND = [["Statistic", "Layer 1", "Layer 2", "value"], Dind, Iind, Cind]
         import csv
 
         with open(OPF, "wb") as f:
```

Comment:
This should fix the other error:
```suggestion
        with open(OPF, "w") as f:
```

Reply:
yes, I had it locally and forgot to change it in the actual PR that I did online, thanks!